collectors/modality-probe-udp-collector/src/lib.rs

```diff
@@ -204,14 +204,30 @@ mod tests {
     use lazy_static::*;
 
     use modality_probe::{
-        compact_log::LogEvent, report::wire::OwnedLogSegment, BulkReporter, ChunkedReporter,
-        LogicalClock, Probe,
+        compact_log::LogEvent, report::wire::OwnedLogSegment, BulkReporter, CausalSnapshot,
+        ChunkedReporter, LogicalClock, Probe, ProbeId,
     };
 
     use modality_probe_collector_common::*;
 
     use super::*;
 
+    struct DefaultableCausalSnapshot(CausalSnapshot);
```

Comment:
Doesn't look like we're using this struct anymore, is it left in intentionally?

Reply:
I accidentally brought this back during a rebase; will remove.